src/user/store/index.ts

```diff
@@ -91,19 +91,19 @@ export class UserStore extends VuexModule {
   }
 
   @Action({ rawError: true })
-  public async setAws(payload: any, condition?: string): Promise<void> {
-    let sync = true
-    Sync.GetSync(payload.username)
+  public async setAws(payload: {user: any, condition?: string, noSync?: boolean}): Promise<void> {
```

Comment:
When testing with this, I found that no matter what `condition` was passed as an argument for `setAws`, it would always be `undefined` once it reached this function.  Apparently, with `Action`s, any arguments beyond the first are discarded.  Loaded `condition` into the payload, along with the `noSync` flag for when the system doesn't want to perform a Pull sync.